parts/kubernetesmastervars.t

```diff
@@ -58,6 +58,11 @@
 {{if AnyAgentHasDisks}}
     "dataStorageAccountPrefixSeed": 97,
 {{end}}
+{{if IsVNETIntegrated}}
+    "allocateNodeCidrs": false,
+{{else}}
+    "allocateNodeCidrs": true,
+{{end}}
```

Comment:
Small nit: Does it need an additional bool or can IsVNETIntegrated be used directly? Or is it because function cannot be used in the yml?